javascripts/discourse/lib/utils.js

```diff
@@ -0,0 +1,12 @@
+export function normalizeName(name) {
```

Comment:
Would be great to extract this to `app/assets/javascripts/discourse/app/lib/utilities.js` in core, maybe you already plan to?

Reply:
TBH, I thought about this, and I'm on the fence. 

Core has similar functions already, like slugify.

This one is different because it won't strip non-Latin chars like Chinese, Cyrillic, etc.

For example: `🇨🇳 Chinese Translation of FAQ 使用守則 中文翻譯` returns `chinese-translation-of-faq-使用守則-中文翻譯``

Do you think this is something worth adding to core?

Reply:
Yes I think so, we would need a few JS unit tests to demonstrate this but I think it could be useful in other cases

Reply:
I moved an improved version of the function to core in https://github.com/discourse/discourse/pull/27805